pennylane_pq/devices.py

```diff
@@ -549,7 +557,7 @@ class ProjectQClassicalSimulator(_ProjectQDevice):
 
     def __init__(self, wires=1, **kwargs):
         kwargs['backend'] = 'ClassicalSimulator'
-        super().__init__(wires=wires, shots=0, **kwargs)
+        super().__init__(wires=wires, shots=1, analytic=True, **kwargs)
```

Comment:
Would it maybe be worth setting more shots as default?

Reply:
Set it to 1024 everywhere :)